gen3/bin/jupyter.sh

```diff
@@ -229,6 +255,14 @@ case "$command" in
   "idle")
     gen3_jupyter_idle_pods "$@"
     ;;
+  "metrics")
+    if [[ $# -gt 0 && "$1" == "upload" ]]; then
+      shift
+      gen3_jupyter_metrics_upload "$@";
+    else
+      gen3_jupyter_metrics "$@"
+    fi
+    ;;
   "prepuller")
```

Comment:
this looks great, Di.  Would you go ahead and add some docs to `doc/jupyter.md`, and a test of metrics fetch to `gen3/test/jupyterTest.sh` too?